build.xml

```diff
@@ -1,45 +0,0 @@
-<project name="user-agent-tools" default="compile" basedir=".">
```

Comment:
Well, it's up to you. As you don't know maven, I don't know ant.

If ant can deal with the new project structure, then you can keep both. Actually, maven could do that but I considered that ant is not needed anymore. I understand if you reject my changes.